charts/portworx/templates/_helpers.tpl

```diff
@@ -25,6 +25,15 @@ release: {{ .Release.Name | quote }}
 {{$version := .Capabilities.KubeVersion.GitVersion | regexFind "^v\\d+\\.\\d+\\.\\d+"}}{{$version}}
 {{- end -}}
 
+{{- define "px.storkKubernetesVersion" -}}
+{{- $version := .Capabilities.KubeVersion.GitVersion | regexFind "^v\\d+\\.\\d+\\.\\d+" -}}
+{{- if semverCompare ">=1.22" $version -}}
+"v1.21.0"
```

Comment:
nit: Lets use v1.21.4, same as what we use in px-installer / spec-gen

P.S.: The [PR](https://github.com/portworx/px-installer/pull/1121) title says it is using 1.21.0, but we ended up using 1.21.4. Will update the title.

Reply:
good catch :) updated